web_hosting_manager/app/constants.js

```diff
@@ -17,6 +24,7 @@ const CONSTANTS = {
       scope: null,
       name: pkg.productName,
       vendor: pkg.author.name,
+      customExecPath: isDevMode ? `${cwd}/node_modules/.bin/electron${isWindows} ${cwd}/app` : app.getPath('exe')
```

Comment:
You'll need to set the `bundle` attribute for OSX platforms, you can check how we've done it in safe-app-plugin or Peruse.

Reply:
addressed in latest commit